src/main/java/com/incognia/feedback/FeedbackEvent.java

```diff
@@ -48,5 +48,11 @@ public enum FeedbackEvent {
   @JsonProperty("password_change_failed")
   PASSWORD_CHANGE_FAILED,
   @JsonProperty("promotion_abuse")
-  PROMOTION_ABUSE;
+  PROMOTION_ABUSE,
+  @JsonProperty("custom_order_flow_fraud")
+  CUSTOM_ORDER_FLOW_FRAUD,
+  @JsonProperty("custom_pos_atm_fraud")
+  CUSTOM_POS_ATM_FRAUD,
+  @JsonProperty("custom_collusion_fraud")
+  CUSTOM_COLLUSION_FRAUD;
```

Comment:
I guess you should point this at https://github.com/inloco/incognia-api-java/pull/78 since these changes are from that PR